package.json

```diff
@@ -4,7 +4,7 @@
   "main": "bin/action.min.js",
   "source": "src/index.ts",
   "devDependencies": {
-    "@actions/core": "^1.2.6",
+    "@actions/core": "^1.10.0",
```

Comment:
I discovered this upgrade won't work on it's own you also have to upgrade microbundle since mircrobundle has a bug bundling the crypo package used in core v1.10. Took me too long to sort out what was happening. see my other comment https://github.com/FirebaseExtended/action-hosting-deploy/pull/261#issuecomment-1535263868